src/HeaderCell.tsx

```diff
@@ -160,10 +161,25 @@ export default function HeaderCell<R, SR>({
   }
 
   function onKeyDown(event: React.KeyboardEvent<HTMLSpanElement>) {
-    if (event.key === ' ' || event.key === 'Enter') {
+    const { key } = event;
+    if (sortable && (key === ' ' || key === 'Enter')) {
       // prevent scrolling
       event.preventDefault();
       onSort(event.ctrlKey || event.metaKey);
+    } else if (
+      resizable &&
+      isCtrlKeyHeldDown(event) &&
+      (key === 'ArrowLeft' || key === 'ArrowRight')
```

Comment:
Open to suggestion on which shortcut to use. I did not find any good example other than [this](https://react-spectrum.adobe.com/react-aria/Table.html#column-resizing). But the example uses different a different strategy where the resizer gets focused. I think we can keep out implementation simple